scalatest/src/main/scala/org/scalatest/tools/RunnerJFrame.scala

```diff
@@ -746,9 +746,9 @@ private[scalatest] class RunnerJFrame(
 
       val itemAction: AbstractAction =
         new AbstractAction(menuItemText) {
-          def actionPerformed(ae: ActionEvent): Unit = {
+          def actionPerformed(ae: ActionEvent | Null): Unit = {
 
-            val checkBox: JCheckBoxMenuItem = ae.getSource().asInstanceOf[JCheckBoxMenuItem]
+            val checkBox: JCheckBoxMenuItem = ae.nn.getSource().asInstanceOf[JCheckBoxMenuItem]
```

Comment:
**Overridden java arguments are never null**
Argument should never be null, immediately remove nullness of parameter.